dataeng/jobs/analytics/EnterpriseEnrollment.groovy

```diff
@@ -0,0 +1,32 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+
+class EnterpriseEnrollment {
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("enterprise-enrollment") {
+            authorization {
+                allVars.get('PROJECT_BASED_ACCESS').each { roles ->
+                    permission(roles.get('ROLE'), roles.get('GITHUB_ID'))
+                }
+            }
+            logRotator common_log_rotator(allVars)
+            parameters common_parameters(allVars)
+            parameters {
+                stringParam('REPORT_DATE', allVars.get('REPORT_DATE'), '')
+                stringParam('EXTRA_ARGS', allVars.get('EXTRA_ARGS'), '')
```

Comment:
So I haven't been including EXTRA_ARGS in any of the jobs, unless it wasn't already set to something. 
And I'm not exactly sure why we have these in the first place, perhaps to support adding parameters for custom runs(failed jobs?), overwrite_n_days etc ? @brianhw do you know more ?

Reply:
Yes, these were added to support adding parameters for custom runs on specific jobs.   Yes, overwriting a value of overwrite_n_days is one of those scenarios, for when we want to run with a value different from the default of 3 (either more or less).  But we hadn't been adding them on all jobs and I don't know offhand why this job would need it.   I suspect it was a cut-and-paste.

Reply:
Although we don't need it right now, but in case we have to run a job with different param so it can be handy & we don't have to manually change the job.